jans-cli-tui/setup.py

```
@@ -6,10 +6,30 @@
 import re
+import time
 from setuptools import setup
 from setuptools import find_packages
 from setuptools.command.install import install
-from urllib.request import urlretrieve
+from urllib.request import urlopen
+from urllib.error import HTTPError, URLError
+
+def _download_with_retry(url, dest, retries=5, delay=10, timeout=60) -> None:
+    if retries <= 0:
+        raise ValueError(f"retries must be a positive integer, got {retries}")
```

Ruff (0.15.21)

[warning] Avoid specifying long messages outside the exception class

(TRY003)